dmcontent/questions.py

```diff
@@ -464,6 +467,9 @@ def get_question(self, field_name):
         if self.id == field_name or field_name in self.fields.values():
             return self
 
+    def unformat_data(self, data):
+        return self.get_data(data)
```

Comment:
Although on the other hand the same can be said about `get_data` and maybe having a similar interface between two methods makes sense.

On the other hand, they iterate over different things: `get_data` iterates over questions and `unformat_data` iterates over data keys.